app/_src/quickstart/docker-demo.md

```diff
@@ -0,0 +1,716 @@
+
+   This isn’t mandatory, but it’s easy to accidentally mix up ports when typing commands manually, leading to unexpected results. If you copy the commands from this guide exactly or know what you’re doing and want to compare results between guides, you can skip this step.
+
+## Prepare the environment
+
+{% if version == "preview" %}
+1. **Retrieve the latest preview version of {{ Kuma }}**
+
+   To use the latest preview version of {{ Kuma }}, retrieve it and export it as an environment variable for later steps:
+
+   ```sh
+   export {{ KUMA_PREVIEW_VERSION }}="$(
+     curl --silent --location {{ url_installer }} \
+       | VERSION="preview" sh -s - --print-version 2> /dev/null
+   )"
+   
+   echo "${{ KUMA_PREVIEW_VERSION }}"
+   ```
+
+   You should see output similar to this:
+
+   ```
+   0.0.0-preview.vabc123def
+   ```
+   {:.no-line-numbers}
+
+   {% capture warning-env-var %}
+   {% danger %}
+   **Warning:** If no output is shown or it looks incorrect, set the `{{ KUMA_PREVIEW_VERSION }}` variable to the correct version manually. Do not continue without this value, as it is needed for later steps. Using the wrong version may cause errors or unexpected problems.  
+   {% enddanger %}
+   {% endcapture %}
+   {{ warning-env-var | indent }}
+   {% endif %}
+
+2. **Install {{ Kuma }}**
+
+   You can download and install {{ Kuma }} using the official installer. The installer automatically detects your operating system (Amazon Linux, CentOS, RedHat, Debian, Ubuntu, macOS) and downloads the appropriate binaries:
+
+   ```sh
+   curl --location {{ url_installer }} {% if version == "preview" or page.edition and page.edition != "kuma" -%}\
+     {% endif %}| VERSION="{{ version_full }}" sh -
+   ```
+
+   To finalize the installation add the {{ Kuma }} binaries to your system’s [PATH](https://en.wikipedia.org/wiki/PATH_(variable)) so the commands are easily accessible:
+
+   ```sh
+   export PATH="$(pwd)/{{ kuma }}-{{ version_full }}/bin:$PATH"
+   ```
+
+3. **Prepare a temporary directory**
+
+   Set up a temporary directory to store resources like data plane tokens, {{ Dataplane }} templates, and logs. Ensure the path does not end with a trailing `/`.
+
+   {% capture warning-colima-adjust-path %}
+   {% warning %}
+   **Important:** If you are using **Colima**, make sure to adjust the path in the steps of this guide. Colima only allows shared paths from the `HOME` directory or `/tmp/colima/`. Instead of `{{ tmp }}`, you can use `{{ tmp-colima }}`.
+   {% endwarning %}
+   {% endcapture %}
+   {{ warning-colima-adjust-path | indent }}
+
+   Check if the directory exists and is empty, and create it if necessary:
+
+   ```sh
+   mkdir -p {{ tmp }}
+   ```
+
+4. **Prepare a Dataplane resource template**
+
+   Create a reusable {{ Dataplane }} resource template for services:
+
+   ```sh
+   echo 'type: Dataplane
```

Comment:
we should add labels to dataplanes like we do in new guide for 2.10.x for universal, and use Dataplane targetRef kind instead of MeshSubset in MeshTrafficPermission later on

Reply:
addressed